pkgs/development/python-modules/onlykey-solo-python/default.nix

```diff
@@ -0,0 +1,34 @@
+{ buildPythonPackage
+, click
+, ecdsa
+, fetchPypi
+, fido2
+, intelhex
+, lib
+, pyserial
+, pyusb
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "onlykey-solo-python";
+  version = "0.0.28";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-Mbi5So2OgeXjg4Fzg7v2gAJuh1Y7ZCYu8Lrha/7PQfY=";
+  };
+
+  propagatedBuildInputs = [ click ecdsa fido2 intelhex pyserial pyusb requests ];
+
+  # no tests
+  doCheck = false;
```

Comment:
Can we add a pythonImportsCheck instead?

Reply:
done